keras_cv/models/backbones/mobilenet_v3/mobilenet_v3_backbone.py

```diff
@@ -421,14 +421,14 @@ def __new__(
                 "input_tensor": input_tensor,
             }
         )
-        return MobileNetV3Backbone.from_preset("mobilenetv3small", **kwargs)
+        return MobileNetV3Backbone.from_preset("mobilenet_v3_small", **kwargs)
 
     @classproperty
     def presets(cls):
         """Dictionary of preset names and configurations."""
         return {
             "mobilenetv3small": copy.deepcopy(
```

Comment:
mobilenet_v3_small, right?

Reply:
Actually this shouldn't be here at all! Miscommunication from the last PR lol